ezyrb/interpolation.py

```diff
@@ -195,31 +179,17 @@ def add_snapshot(self, new_mu=None, snapshot_file=None):
 		:param snapshot_file string
```

Comment:
Pod, Interpolation and Cvt classes will change in next PR, so I would write documentation/examples/tutorial only for the finished design (and avoid to do it two times).
Do you think is a reasonable idea? Maybe is it better merge everything in this pull request? (imho, merging can create more confusion for code review)